src/clocksi_vnode.erl

```diff
@@ -125,13 +125,13 @@ abort(ListofNodes, TxId) ->
 init([Partition]) ->
     PreparedTx = ets:new(list_to_atom(atom_to_list(prepared_tx) ++
                                           integer_to_list(Partition)),
-                         [set, {write_concurrency, true}]),
+                         [set]),
     CommittedTx = ets:new(list_to_atom(atom_to_list(committed_tx) ++
                                            integer_to_list(Partition)),
-                          [set, {write_concurrency, true}]),
+                          [set]),
     ActiveTxsPerKey = ets:new(list_to_atom(atom_to_list(active_txs_per_key)
                                            ++ integer_to_list(Partition)),
-                              [bag, {write_concurrency, true}]),
+                              [bag]),
```

Comment:
From the docs:
{write_concurrency,boolean()} Performance tuning. Default is false, in which case an operation that mutates (writes to) the table will obtain exclusive access, blocking any concurrent access of the same table until finished. If set to true, the table is optimized towards concurrent write access. Different objects of the same table can be mutated (and read) by concurrent processes. This is achieved to some degree at the expense of memory consumption and the performance of sequential access and concurrent reading.

These tables are accessed only by this vnode which is single threaded.  Having write concurrency set to true makes it confusing because it seems like it should be accessed by several concurrent writers.  Also it means more memory consumption and slower access.